Modules/Sources/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -676,6 +683,11 @@ public enum ServerSidePaymentCaptureError: Error, LocalizedError {
     }
 }
 
+//periphery:ignore - logging this error detail in WooCommerce is useful, if it ever happens. It's part of the public API here.
+public enum CardPresentPaymentStoreError: Error {
```

Comment:
nit: this error enum doesn't seem used outside of Yosemite. Could it be internal to avoid the Periphery warning?

```suggestion
enum CardPresentPaymentStoreError: Error {
```

Reply:
It's not used, but I see it as part of the public API, as the store is used from outside Yosemite and the clients could definitely care about the errors that come back.